plugins/modules/ovirt_disk.py

```diff
@@ -731,6 +733,8 @@ def main():
 
             # Upload disk image in case it's new disk or force parameter is passed:
             if module.params['upload_image_path'] and (is_new_disk or module.params['force']):
+                if module.params['format'] == 'cow' and module.params['content_type'] == 'iso':
+                    module.warn("Parameter 'format' should not be 'cow' when uploading iso image, please change it to 'raw'.")
```

Comment:
Please rephrase: `To upload an ISO image 'format' parameter needs to be set to 'raw'`